routes/products.js

```diff
@@ -2,6 +2,7 @@ const express = require("express");
 const productsRouter = express.Router();
 
 const { getProductById, getAllProducts, createProduct } = require("../db");
+const ordersRouter = require("./orders");
```

Comment:
What do we use ordersRouter for in products.js?

Reply:
woops didnt even see that there i removed it..